entwatch/ze_eternal_journey_go_b2.cfg

```diff
@@ -1,200 +0,0 @@
-		"buttonclass"       "func_button"
-		"filtername"        "zmfire_player"
-		"hasfiltername"     "true"
-		"blockpickup"       "false"
-		"allowtransfer"     "false"
-		"forcedrop"         "false"
-		"chat"              "true"
-		"hud"               "true"
-		"hammerid"          "18790442"
-		"mode"              "2"
-		"maxuses"           "0"
-		"cooldown"          "60"
-		"maxamount"         "1"
-	}
-	"6" {
-		"name"              "Zombie Gravity"
-		"shortname"         "ZM Gravity"
-		"color"             ""
-		"buttonclass"       "func_button"
-		"filtername"        "zmgravity_player"
-		"hasfiltername"     "true"
-		"blockpickup"       "false"
-		"allowtransfer"     "false"
-		"forcedrop"         "false"
-		"chat"              "true"
-		"hud"               "true"
-		"hammerid"          "18791994"
-		"mode"              "2"
-		"maxuses"           "0"
-		"cooldown"          "60"
-		"maxamount"         "1"
-	}
-	"7" {
-		"name"              "Zombie Freeze"
-		"shortname"         "ZM Freeze"
-		"color"             ""
-		"buttonclass"       "func_button"
-		"filtername"        "zmfreeze_player"
-		"hasfiltername"     "true"
-		"blockpickup"       "false"
-		"allowtransfer"     "false"
-		"forcedrop"         "false"
-		"chat"              "true"
-		"hud"               "true"
-		"hammerid"          "18805946"
-		"mode"              "2"
-		"maxuses"           "0"
-		"cooldown"          "60"
-		"maxamount"         "1"
-	}
-	"8" {
-		"name"              "Zombie Skin 1"
-		"shortname"         "ZM Skin 1"
-		"color"             ""
-		"buttonclass"       "func_button"
-		"filtername"        "zmskin1_player"
-		"hasfiltername"     "true"
-		"blockpickup"       "false"
-		"allowtransfer"     "false"
-		"forcedrop"         "false"
-		"chat"              "true"
-		"hud"               "true"
-		"hammerid"          "18808768"
-		"mode"              "1"
-		"maxuses"           "0"
-		"cooldown"          "0"
-		"maxamount"         "1"
-	}
-	"9" {
-		"name"              "Zombie Skin 2"
-		"shortname"         "ZM Skin 2"
-		"color"             ""
-		"buttonclass"       "func_button"
-		"filtername"        "zmskin2_player"
-		"hasfiltername"     "true"
-		"blockpickup"       "false"
-		"allowtransfer"     "false"
-		"forcedrop"         "false"
-		"chat"              "true"
-		"hud"               "true"
-		"hammerid"          "18809475"
-		"mode"              "1"
-		"maxuses"           "0"
-		"cooldown"          "0"
-		"maxamount"         "1"
-	}
-	"10" {
-		"name"              "Ultimate"
-		"shortname"         "Ultimate"
-		"color"             ""
-		"buttonclass"       "func_button"
-		"filtername"        "ultimate_player"
-		"hasfiltername"     "true"
-		"blockpickup"       "false"
-		"allowtransfer"     "true"
-		"forcedrop"         "true"
-		"chat"              "true"
-		"hud"               "true"
-		"hammerid"          "19321023"
-		"mode"              "4"
-		"maxuses"           "1"
-		"cooldown"          "15.5"
-		"maxamount"         "1"
-	}
-}
```

Comment:
No reason to delete a config just because we don't have the map. Rather have it available if we end up adding the map in the future.